src/lib/tasks/server/create_pipeline_task.cpp

```diff
@@ -0,0 +1,49 @@
+#include "create_pipeline_task.hpp"
+
+#include <boost/algorithm/string.hpp>
+
+#include "sql/sql_pipeline.hpp"
+
+namespace opossum {
+
+void CreatePipelineTask::_on_execute() {
+  auto result = std::make_unique<CreatePipelineResult>();
+
+  try {
+    result->sql_pipeline = std::make_shared<SQLPipeline>(_sql);
```

Comment:
This looks like something that should be trivial to execute. Why does this have its own task?

Reply:
This goes for other tasks as well.

Reply:
This is a separate task because we wanted to reduce the computational load on the server main thread as much as possible. If we receive a large SQL string, parsing can take a bit (not a lot but you know) so we thought it made sense to move this to a task. (if I remember correctly, this was even your idea)